src/connect.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+import assert from './util/assert'
+import isObservable from './util/isObservable'
+import isDispatcher from './util/isDispatcher'
+
```

Comment:
For the sake of JSDoc or ESDoc:

```
/**
 * Operations on the observable to select a prop for connect
 * @callback selector
 * @param {Observable} observable to operate on
 * @return {Observerable} resulting observable
 */
```

Clearly that could use better description. Helped me to understand the purpose here to know what the selector required.